### ISSUE TITLE
Submit button disappears in chat

### ISSUE THREAD

**Issue by js** [helpscout/beacon-ios-sdk#139]
In the 2.1.2 SDK on iOS 14 the submit button disappears from the chat input textview as soon as something is entered into the textfield above the keyboard. 

Looking in the view debugger it seems the button is still there (and it can be tapped) but the background changes to white whenever a character is entered into the textfield, which also has a white background.

**Comment by ScottPetit** [comment]
Hey @js thanks for the report.  This seems to be an oversight on our part for Beacons that use white as their Beacon color.  We're tracking it [internally here](https://helpscout.atlassian.net/browse/IOSSDK-442) and will let you know when we have a fix.

In the meantime you can change the color your Beacon uses by setting the `color` property of your `HSBeaconSettings` object.  That would look something like: 

```swift
let settings = HSBeaconSettings(beaconId: beaconId)
settings.color = UIColor.black
```

**Comment by codenameDuchess** [comment]
@js 👋 Thanks for reporting this issue. We wanted to let you know we fixed this in the latest version (2.2.0). You can read more about what's in this release in our [Changelog](https://github.com/helpscout/beacon-ios-sdk/blob/master/CHANGELOG.md).
